themes/jquery/header.php

```diff
@@ -1,11 +1,11 @@
+<?php jq_content_security_policy() ?>
 <!doctype html>
 <html class="no-js" <?php language_attributes(); ?>>
 <head>
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 
 	<title><?php
-		global $page, $paged;
```

Comment:
This was a carryover from the default Wordpress header.php, but these variables are not used.